controllers/annotation/cesExposedPortAnnotator.go

```diff
@@ -0,0 +1,67 @@
+package annotation
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/cloudogu/cesapp-lib/core"
+	corev1 "k8s.io/api/core/v1"
+)
+
+type CesExposedPort struct {
+	Protocol   string `json:"protocol"`
+	Port       int    `json:"port"`
+	TargetPort int    `json:"targetPort"`
+}
+
+const CesExposedPortAnnotation = "k8s-dogu-operator.cloudogu.com/ces-exposed-ports"
+
+// CesExposedPortAnnotator collects ces service information and annotates them to a given K8s service.
+type CesExposedPortAnnotator struct{}
```

Comment:
hmm you're right... I don't really understand, why it is the way it is... but it should be the same on all annotators. Maybe we can discuss with another colleague about it or leave it as is.

Reply:
We discussed that we should keep the structure as it is. The restructuring would not have such great  value. In addition, the annotator could possibly be expanded so that this structure is still helpful.